project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject puppetlabs/jruby-utils "5.1.5-SNAPSHOT"
+(defproject puppetlabs/jruby-utils "5.2.0-SNAPSHOT"
```

Comment:
@justinstoller is it okay to bump the minor version here?

Reply:
Yes